main.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const ball = document.querySelector('.ball');
+
+document.addEventListener('click', event => {
+const ballStyles = getComputedStyle(ball);
+const width = ballStyles.width.replace('px', '');
+const height = ballStyles.height.replace('px', '');
```

Comment:
there is a better way to get dimensions https://developer.mozilla.org/ru/docs/Web/API/Element/getBoundingClientRect